projects/components/src/table/table.component.ts

```diff
@@ -234,6 +241,9 @@ export class TableComponent
   @Output()
   public readonly toggleAllChange: EventEmitter<boolean> = new EventEmitter<boolean>(); // True: expand, False: collapse
 
+  @Output()
+  public readonly pageChange: EventEmitter<boolean> = new EventEmitter<boolean>();
```

Comment:
Why a boolean? I'd think this would emit the new page index - corresponding to `page` the input, which we're not currently exposing.

Reply:
makes sense. emitting the entire page event now